game-core/src/main/java/games/strategy/triplea/settings/GameSetting.java

```diff
@@ -40,7 +41,7 @@
    * default value.
    */
   default T getValueOrThrow() {
-    return getValue().get();
+    return getValue().orElseThrow(() -> new NoSuchElementException());
```

Comment:
You should be able to use the 0-argument variant of this method:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#orElseThrow()

Reply:
Done.